### ISSUE TITLE
Website is slow though I am using Full version of Zen Cache

### ISSUE THREAD

**Issue by mahendrasengar** [wpsharks/comet-cache#466]
Hi,

I am using full upgraded version of Zen Cache pro for my website setmyoffer.com. It seems the plugin is not working fine for me. It perhaps creates cache everytime I open website is opened. Once the website is open it is fast for few minutes but If I open it again after sometime without clearing cache It again takes a lot of time to open. I have done all the settings as recommended except for separate cache for all the users. But website is slow even if I am same user is opening though the resources have been cached previously.

**Comment by raamdev** [comment]
@mahendrasengar I just reviewed your site and ZenCache appears to be working properly. I did notice that you have _a lot_ of static resources being loaded on your home page--there were 630 requests according to Google Chrome when I visited your home page.

ZenCache is a page-caching plugin, which means it increases performance by reducing the need to query the WordPress database on every page load. ZenCache can also decrease the overall file size being loaded by compressing and minifying the HTML, CSS, and JavaScript (HTML Compression). 

However, if you have a large number of static resources being loaded, you'll need to also use a CDN such as [Amazon CloudFront](http://zencache.com/r/amazon-cloudfront/) or [MaxCDN](http://zencache.com/r/maxcdn/) or [CDN77](http://zencache.com/r/cdn77/). ZenCache supports all of these CDNs. See [Introduction to Static CDN Filters](http://zencache.com/kb-article/introduction-to-static-cdn-filters/) for more information.
